boardswarm-cli/src/main.rs

```diff
@@ -35,6 +35,44 @@ mod ui;
 mod ui_term;
 mod utils;
 
+#[derive(Clone, Copy)]
+struct ItemTypes(pub ItemType);
+
+impl From<ItemTypes> for ItemType {
+    fn from(val: ItemTypes) -> Self {
+        match val {
+            ItemTypes(t) => t,
+        }
+    }
+}
+
+impl std::fmt::Debug for ItemTypes {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let t: ItemType = (*self).into();
+        write!(f, "{:?}", t)
```

Comment:
Or potentially even better just directly call out to the wrapped types implementation so you avoid a potential useless str conversion 
```suggestion
        self.0.fmt(f)
```